### PR TITLE
fix(client): fix modal heading CSS for light theme

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to: #40071 

https://github.com/freeCodeCamp/freeCodeCamp/pull/40071#issuecomment-716737906

The use of the `!important` flag is only necessary because of the mentioned PR (the night-mode styling overrides the modal, when it is used outside of `/settings`)

### CODE REVIEW

File: client/src/components/layouts/global.css [hunk 1]
```diff
@@ -436,7 +436,7 @@ pre {
 
 .panel-primary > .panel-heading {
   background-color: transparent;
-  border-color: var(--color-primary);
+  border-color: var(--primary-color);
 }
```
Comment (ahmadabdolsaheb) [inline client/src/components/layouts/global.css:440]: We could just add ` color: var(--primary-color);` here and we would not need the .panel-heading style in solution-viewer.css

Comment (Sky020) [reply client/src/components/layouts/global.css:440]: @ahmadabdolsaheb Seeing as `solution-viewer.css` is such a small file, should I just delete it, and move the CSS to `global.css`?

Comment (ahmadabdolsaheb) [reply client/src/components/layouts/global.css:440]: yes please :)

File: client/src/components/layouts/global.css [hunk 2]
```diff
@@ -436,7 +436,17 @@ pre {
 
 .panel-primary > .panel-heading {
   background-color: transparent;
-  border-color: var(--color-primary);
+  border-color: var(--primary-color);
+  color: var(--primary-color) !important;
```
Comment (ahmadabdolsaheb) [inline client/src/components/layouts/global.css:440]: Do we still need the !important?

Comment (Sky020) [reply client/src/components/layouts/global.css:440]: No, it does not appear necessary anymore.
```suggestion
  color: var(--primary-color);
```

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**